tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -1101,6 +1101,12 @@ platform_tests/test_intf_fec.py:
     conditions:
       - "asic_type in ['vs']"
 
+platform_tests/test_intf_fec.py::test_verify_fec_histogram:
+  skip:
```

Comment:
Please use xfail instead of skip.

Reply:
Hi @yutongzhang-microsoft , it's not urgent to merge this PR. We can wait for some feedback on the bug firstly.

Reply:
all tests are passing on our side, do you mind to help check the latest status again?

<img width="1033" height="192" alt="image" src="https://github.com/user-attachments/assets/654973fd-a222-4b99-9d53-fc086f8c92b0" />